### PR TITLE
Added page visit counter styling and functionality

### PR DESCRIPTION
**Pull request by Ericsheoran**
- Added a visit counter to the footer section.
- Styled the counter to align with the website's design.
- Implemented functionality to track visits per page using localStorage.

### CODE REVIEW

**Comment by Ericsheoran** [comment]
https://youtu.be/o7cE6cIac5g

File: home/templates/pages/index.html
Comment (jbm1412) [inline home/templates/pages/index.html:1]: Screenshot (1996)

Please fix this issue

Comment (jbm1412) [reply home/templates/pages/index.html:1]: Please stick to this template 

Screenshot (1997)

**Review by jbm1412 — CHANGES_REQUESTED** [review]
https://github.com/user-attachments/assets/c46962df-5b59-4bcf-88e2-c750f7f0d023

It's not working, please check your js file and script.

**Review by jbm1412 — CHANGES_REQUESTED** [review]
Screenshot (2023)

Tested -> Works. Looks Good, but fix the conflicting files

**Review by jbm1412 — CHANGES_REQUESTED** [review]
Screenshot (2026)

Please fix this issue, check and see before pushing it

**Review by jbm1412 — APPROVED** [review]
Screenshot (2034)

Tested -> Works. Looks Good!

**Pull request merged**